src/View/Requirements_Backend.php

```diff
@@ -295,7 +298,7 @@ public function getCombinedFilesFolder()
         return Config::inst()->get(__CLASS__, 'default_combined_files_folder');
     }
 
-    /**
```

Comment:
Um! Trying not to be pedantic here (I lie I am), but `/**` is required for phpdoc. :)